app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -153,7 +148,7 @@ class BackgroundServices(
             }
 
             if (authType == AuthType.Signup) {
-                context.components.analytics.metrics.track(Event.FXANewSignup)
+                context.components.analytics.metrics.track(Event.FXNewSignUp)
```

Comment:
(not your commit, but...) ugh, inconsistent naming!

Reply:
Yes, I will improve it in a follow up

Reply:
Done!